ThingIFSDK/ThingIFSDK/ThingIFAPI.swift

```diff
@@ -124,6 +125,7 @@ public class ThingIFAPI: NSObject, NSCoding {
     (Specified on creation of ThingIFAPI instance.)
     When you're sure that the on board process has been done,
     this method is convenient.
+     If you are using a gateway, you need to use onboardEndnodeWithGateway instead.
```

Comment:
better to describe like this: 
If you are using a gateway, you need to use onboardEndnodeWithGateway to onboard endnode instead.